### ISSUE TITLE
PyPI Namespace Issue with GeoCAT-f2py

### ISSUE THREAD

**Issue by erogluorhan** [NCAR/geocat-comp#212]
**Description**
When GeoCAT-comp is installed via PyPI (i.e. `pip install geocat-comp`), importing  GeoCAT-f2py functions through GeoCAT-comp gives errors, e.g. `from geocat.comp import rgrid2rcm`

**To Reproduce**
1. `conda create --name geocat python=3.9`
2. `pip install geocat-comp`
3. Go to your IDE, e.g. in PyCharm, choose your interpreter to be the "geocat" environment you just created 
4. In Python Console or in a Python script, try running `from geocat.comp import rgrid2rcm`
5. See error.

**Output**
```
Traceback (most recent call last):
  File "<input>", line 1, in <module>
ImportError: cannot import name 'rgrid2rcm' from 'geocat.comp' (/Users/oero/miniconda3/envs/geocat-pip/lib/python3.9/site-packages/geocat/comp/__init__.py)

```

**OS:**
MacOS 11.6.2

**Environment**
```
# Name                    Version                   Build  Channel
appdirs                   1.4.4                    pypi_0    pypi
bzip2                     1.0.8                h0d85af4_4    conda-forge
ca-certificates           2021.10.8            h033912b_0    conda-forge
certifi                   2021.10.8                pypi_0    pypi
cf-xarray                 0.7.0                    pypi_0    pypi
cftime                    1.6.0                    pypi_0    pypi
charset-normalizer        2.0.12                   pypi_0    pypi
cloudpickle               2.0.0                    pypi_0    pypi
cycler                    0.11.0                   pypi_0    pypi
dask                      2022.3.0                 pypi_0    pypi
eofs                      1.4.0                    pypi_0    pypi
fonttools                 4.31.2                   pypi_0    pypi
fsspec                    2022.2.0                 pypi_0    pypi
geocat-comp               2022.3.0                 pypi_0    pypi
geocat-f2py               2022.3.1                 pypi_0    pypi
idna                      3.3                      pypi_0    pypi
kiwisolver                1.4.0                    pypi_0    pypi
libffi                    3.4.2                h0d85af4_5    conda-forge
libzlib                   1.2.11            h9173be1_1013    conda-forge
locket                    0.2.1                    pypi_0    pypi
matplotlib                3.5.1                    pypi_0    pypi
metpy                     1.2.0                    pypi_0    pypi
ncurses                   6.3                  he49afe7_0    conda-forge
numpy                     1.22.3                   pypi_0    pypi
openssl                   1.1.1n               h6c3fc93_0    conda-forge
packaging                 21.3                     pypi_0    pypi
pandas                    1.4.1                    pypi_0    pypi
partd                     1.2.0                    pypi_0    pypi
pillow                    9.0.1                    pypi_0    pypi
pint                      0.18                     pypi_0    pypi
pip                       22.0.4             pyhd8ed1ab_0    conda-forge
pooch                     1.6.0                    pypi_0    pypi
pyparsing                 3.0.7                    pypi_0    pypi
pyproj                    3.3.0                    pypi_0    pypi
python                    3.9.12          h8b4d769_0_cpython    conda-forge
python-dateutil           2.8.2                    pypi_0    pypi
python_abi                3.9                      2_cp39    conda-forge
pytz                      2022.1                   pypi_0    pypi
pyyaml                    6.0                      pypi_0    pypi
readline                  8.1                  h05e3726_0    conda-forge
requests                  2.27.1                   pypi_0    pypi
scipy                     1.8.0                    pypi_0    pypi
setuptools                60.10.0          py39h6e9494a_0    conda-forge
six                       1.16.0                   pypi_0    pypi
sqlite                    3.37.1               hb516253_0    conda-forge
tk                        8.6.12               h5dbffcc_0    conda-forge
toolz                     0.11.2                   pypi_0    pypi
traitlets                 5.1.1                    pypi_0    pypi
tzdata                    2022a                h191b570_0    conda-forge
urllib3                   1.26.9                   pypi_0    pypi
wheel                     0.37.1             pyhd8ed1ab_0    conda-forge
xarray                    2022.3.0                 pypi_0    pypi
xz                        5.2.5                haf1e3a3_1    conda-forge
zlib                      1.2.11            h9173be1_1013    conda-forge
```

**Comment by marodrig** [comment]
I found a stack [overflow](https://stackoverflow.com/questions/39811929/package-installed-by-conda-python-cannot-find-it) discussion that might be helpful.

**Comment by anissa111** [comment]
Resolved in v2023.06.0
